docs/tutorials/notebooks/Monge_Gap.ipynb

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
<p>... instantiated for the squared Euclidean cost.</p><p>... that will be bot -&gt; ... that will be both</p><p> </p><p>Please use numbered lists instead of unnumbered lists + (i)/(ii).</p><p>The rendering of "... class. For its instantiation, we need ..." seems to have strike-through enabled.</p><p>Please add link to the MLP class as {class}<code>...</code>.</p><p> </p><p> </p><p>TODO(michalk8): add a section header for <code>MapEstimator</code> and use it when referring to it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/361/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>